onnxruntime/core/providers/rocm/nn/pool.cc

```diff
@@ -0,0 +1,353 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/providers/shared_library/provider_api.h"
+#include "core/providers/rocm/miopen_common.h"
+#include "core/providers/rocm/nn/pool.h"
+#include "core/providers/rocm/nn/max_pool_with_index.h"
+#include "core/providers/rocm/math/unary_elementwise_ops_impl.h"
```

Comment:
Could you please sort the header file?

Reply:
No, move the header will cause additional compilation errors to pop up. It seem some of the included headers are not self contained due to historical issue. I don't want to touch them headers to fix it.